src/services/auth-service/auth-service.ts

```diff
@@ -20,6 +20,8 @@ class AuthService implements IAuthService {
 
     login(user: User, accessToken: string) {
         this.user(user);
+        console.log(user);
+
         this.accessToken(accessToken);
 
         localStorageService.setItem(StorageKeys.User, JSON.stringify(user));
```

Comment:
```suggestion
        localStorageService.setItem(StorageKeys.User, user);
```
sorry, I missed during previous reviews, please remove JSON.stringify
cause all serialization and parsing is done internally in the StorageService

Reply:
we can not remove  JSON.stringify because this function accept login(user: User, accessToken: string)  and setItem want to accept string

Reply:
ah, just checked storage service and noticed that is not typed correctly

here is correct example
```
import { IStorageService, StorageKey } from './Storage.types';

class StorageService implements IStorageService {
  constructor(private readonly storage: Storage) {}

  setItem<T>(key: StorageKey, item: T) {
    this.storage.setItem(key, JSON.stringify(item));
  }

  getItem<T>(key: StorageKey): T | null {
    const item = this.storage.getItem(key);
    if (item) {
      return JSON.parse(item);
    }
    return null;
  }

  removeItem(key: StorageKey) {
    this.storage.removeItem(key);
  }
}

export const localStorageService = new StorageService(localStorage);
export const sessionStorageService = new StorageService(sessionStorage);
```

class itself shouldn't be generic, methods should be generic, @RC-3222 missed it during implementation

Reply:
Alexander fixed it